### ISSUE TITLE
Fresh install, 'Continue on inspect.dev' signin doesn't do anything

### ISSUE THREAD

**Issue by ryanhalley** [inspectdev/inspect-issues#154]
I just installed the Windows version, saw the button to continue on the website, clicked it, taken back to website where I'm asked to login to download, which I'd already done to be able to download app. (I can access the Account area so I'm still logged in) Can't proceed further - why does app not think I'm logged in.

**Comment by auchenberg** [comment]
Hi @ryanhalley 

You should be redirected to http://inspect.dev/auth/desktop which will trigger a url handler to open the Inspect app on your computer. 

Which browser are you using? Try to visit http://inspect.dev/auth/desktop in a Chromium based browser.

**Comment by ryanhalley** [comment]
I'm using Chrome latest version on PC and get directed here
https://inspect.dev/login?action=%2Fauth%2Fdesktop - following your link
redirects to that too.

I tried to logout in Account, to see if that would wake it up but clicking Logout doesn't give any feedback I've been logged out, yet if I return to the home page I have been logged out, so that redirect isn't working for me either.

**Comment by auchenberg** [comment]
Interesting, are you blocking any kind of cookies in your Chrome browser?

Once you login via https://inspect.dev/login you should be logged in for the next 30 days, and when visiting https://inspect.dev/account you should see your account information,

Does that happen?

**Comment by ryanhalley** [comment]
Once I logged in I was able to get to https://inspect.dev/account and when returning to the page I was still logged in. I have Adguard installed so potentially it was blocking one of your cookies, not sure. It was weird how the site wasn't able to present a logged out state so I think you've got a bug there, something's not right. Would be better if you just implement the login via the main app and avoid needing to use a website too.
